page03_contact.php

```diff
@@ -2,117 +2,107 @@
+		<div class="w3-black">
+			<div class="w3-container w3-content w3-padding-64" style="max-width:100%">
+
+
+				<div class="w3-content w3-container w3-padding-32 w3-display-middle-large w3-black w3-image">
+
+					<?php
+					//Get the connection to the database in phpMyAdmin
+					require_once("communityproject_mysqli.php");
+					?>
+
+					<!-- Form -->
+					<form action="connect.php" method="post" class="w3-container w3-card-4 w3-black w3-text-orange w3-margin"> 
+						<h2 class="w3-center">CONTACT US</h2>
+
+						<div class="w3-row w3-section">
+							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-user"></i></div>
+							<div class="w3-rest">
+								<input class="w3-input w3-border" id="name" name="name" type="text" placeholder= "Name">
+							</div>
+						</div>		
+
+						<div class="w3-row w3-section">
+							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-envelope-o"></i></div>
+							<div class="w3-rest">
+								<input class="w3-input w3-border" id="email" name="email" type="text" placeholder="Email">
+							</div>
+						</div>
+
+						<div class="w3-row w3-section">
+							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-phone"></i></div>
+							<div class="w3-rest">
+								<input class="w3-input w3-border" id="phone" name="phone" placeholder="A contact number" >
+							</div>
+						</div>
+
+						<div class="w3-row w3-section">
+							<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-pencil"></i></div>
+							<div class="w3-rest">
+								<input class="w3-input w3-border" id="message" name="message" type="text" placeholder="Message">
+							</div>
+						</div>
+
+						<p class="w3-center">
+							<input class="w3-button w3-section w3-orange w3-ripple" type="submit" value="Submit"> 
+						</p>
+					</form>
 
-			<div class="w3-row w3-section">
-		  		<div class="w3-col" style="width:50px"><i class="w3-xxlarge fa fa-pencil"></i></div>
-					<div class="w3-rest">
-			  			<input class="w3-input w3-border" id="message" name="message" type="text" placeholder="Message">
 				</div>
-			</div>
 
-			<p class="w3-center">
-			<input class="w3-button w3-section w3-blue w3-ripple" type="submit" value="Submit"> 
-			</p>
-			</form>
-			
+				<br>
+				<br>
+				<br>
+
+			</div>
 		</div>
-		
-		<br>
-		<br>
-		<br>
-		<br>
-		<br>
-		<br>
 
-	
 
 		<!-- Footer -->
-	<footer class="w3-bottom footer w3-center w3-black w3-padding-16">
-	<p1>Created by <a target="_blank" class="w3-hover-text-orange">Soundwave</a> &copy; Copyright Liv Amer 2022</p1>
-	<br>
-	<br>
- 	<a href="#top" class="back-to-top-link w3-button w3-light-grey"><i class="fa fa-arrow-up w3-margin-right"></i>To the top</a>
-	</footer>
-		
+		<footer class="w3-bottom w3-center w3-grey w3-padding-16">
+			<p>Created by <a target="_blank" class="w3-hover-text-orange">Soundwave</a> &copy; Copyright Liv Amer 2022</p>
+			<a href="#top" class="w3-button w3-black">To the top </a>
```

Comment:
back to top button connected; although user does not need to scroll on laptops but helpful for those on smaller screens